error 'NotImplementedError: Column or row spanning cells are not implemented.' when building doc as text

```
PYTHONPATH=. http_proxy='127.0.0.1:9' sphinx-build -N -btext doc/ \
	doc/_build/text # text generator
Running Sphinx v1.7.5
making output directory...
loading pickled environment... not yet created
building [mo]: targets for 0 po files that are out of date
building [text]: targets for 1 source files that are out of date
updating environment: 1 added, 0 changed, 0 removed
reading sources... [100%] index

looking for now-outdated files... none found
pickling environment... done
checking consistency... done
preparing documents... done
writing output... [100%] index
/build/lua-unit-3.3/doc/index.rst:1737: WARNING: undefined label: minuszero (if the link has no caption the label must precede a section header)
/build/lua-unit-3.3/doc/index.rst:1747: WARNING: undefined label: minuszero (if the link has no caption the label must precede a section header)
/build/lua-unit-3.3/doc/index.rst:1757: WARNING: undefined label: minuszero (if the link has no caption the label must precede a section header)

Exception occurred:
  File "/usr/lib/python3/dist-packages/sphinx/writers/text.py", line 627, in visit_entry
    raise NotImplementedError('Column or row spanning cells are '
NotImplementedError: Column or row spanning cells are not implemented.
The full traceback has been saved in /tmp/sphinx-err-21uy8cat.log, if you want to report the issue to the developers.
Please also report this if it was a user error, so that a better error message can be provided next time.
A bug report can be filed in the tracker at <https://github.com/sphinx-doc/sphinx/issues>. Thanks!
make[1]: *** [debian/rules:12: override_dh_auto_build] Error 2
```

--- a/doc/index.rst
+++ b/doc/index.rst
@@ -1256,20 +1256,31 @@ that a value is anything but the boolean *true* or *false*.
 
 The below table sums it up:
 
-============  ============  ===================  ================  =============  ===================  ===============
-**True assertion family**                                            **False assertion family**
------------------------------------------------------------------  ---------------------------------------------------
-Input Value   assertTrue()   assertEvalToTrue()  assertNotFalse()  assertFalse()  assertEvalToFalse()  assertNotTrue()
-============  ============  ===================  ================  =============  ===================  ===============
-*true*        OK            OK                   OK                Fail           Fail                 Fail
-*false*       Fail          Fail                 Fail              OK             OK                   OK 
-*nil*         Fail          Fail                 OK                Fail           OK                   OK 
-*0*           Fail          OK                   OK                Fail           Fail                 Fail 
-*1*           Fail          OK                   OK                Fail           Fail                 Fail
-*"hello"*     Fail          OK                   OK                Fail           Fail                 Fail
-============  ============  ===================  ================  =============  ===================  ===============
+    **True assertion family**
 
+============  ============  ===================  ================
+Input Value   assertTrue()  assertEvalToTrue()   assertNotTrue()
+============  ============  ===================  ================
+*true*        OK            OK                   OK
+*false*       Fail          Fail                 Fail
+*nil*         Fail          Fail                 OK
+*0*           Fail          OK                   OK
+*1*           Fail          OK                   OK
+*"hello"*     Fail          OK                   OK
+============  ============  ===================  ================
 
+    **False assertion family**
+
+============  ================  =============  ===================
+Input Value   assertNotFalse()  assertFalse()  assertEvalToFalse()
+============  ================  =============  ===================
+*true*        Fail              Fail           Fail
+*false*       OK                OK             OK
+*nil*         Fail              OK             OK
+*0*           Fail              Fail           Fail
+*1*           Fail              Fail           Fail
+*"hello"*     Fail              Fail           Fail
+============  ================  =============  ===================
 
 .. function:: assertEvalToTrue(value [, extra_msg])
 

